modules/RTC/BridgeChannel.js

```diff
@@ -418,7 +418,10 @@ export default class BridgeChannel {
                 }
             }
 
-            if (!this._closedFromClient) {
+            // When the JVB closes the connection gracefully due to the participant being alone in
+            // the meeting it uses code 1001, so treat that as a graceful close and don't say
+            // anything.
+            if (!this._closedFromClient && event.code !== 1001) {
```

Comment:
hmm and it's really okay not to emit `RTCEvents.DATA_CHANNEL_CLOSED` and not adjust `this._connected = false;` ?

Reply:
Good point Pawel! Updated, can you PTAL?